backend/dal/offerDAL.go

```diff
@@ -43,7 +43,7 @@ func (dal *OfferDAL) GetLastOffers(before time.Time, size int) ([]models.Offer,
 }
 
 // InsertOfferInDB Receives an offer as a parameter and inserts into the database
-func (dal *OfferDAL) InsertOfferInDB(offer *models.Offer) error {
+func (dal *OfferDAL) InsertOfferInDB(offer *models.Offer) (string, error) {
```

Comment:
It would be nice to update the comment description of the function explaining the return parameters.

Reply:
Resolved in commit 222813e